### PR TITLE
lxqt: init at 0.11

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

Add [LXQt](http://lxqt.org/) (The Lightweight Qt Desktop Environment) to nixpkgs.
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by joachifm** [comment]
Wow, this is a pretty huge addition :) Looks pretty good to me.

**Comment by romildo** [comment]
I have tested on a VM using a minimal configuration and it works for me. As I am not yet using it extensively some issues may appear. Once merged it will be easier to do more extensive tests.

The LXQt optional packages (besides `qterminal`) will be added after merging.

**Comment by abbradar** [comment]
Nice! I had once an attempt to bring LXQt to NixOS but was somewhat disappointed in the end (I think multi-monitor support was somehow broken at that time) and abandoned it.

Two small issues from 5 minute testing in a VM:

* No start menu icon (is this perhaps intended?);
* `lxqt-config-monitor` doesn't work.

**Comment by romildo** [comment]
@abbradar I have fixed the missing panel icon for the main menu plugin.

**Comment by romildo** [comment]
@abbradar I have also fixed the launching of the _Monitor Settings_ dialog (`lxqt-config-monitor`).

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -13702,8 +13702,6 @@ in
 
   pcmanfm = callPackage ../applications/misc/pcmanfm { };
 
-  pcmanfm-qt = callPackage ../applications/misc/pcmanfm-qt { };
-
```
Comment (dezgeg) [inline pkgs/top-level/all-packages.nix]: Please add `pcmanfm-qt = lxde.pcmanfm-qt;` to keep old user configs working.

Comment (romildo) [inline pkgs/top-level/all-packages.nix]: @dezgeg Done.

**Comment by bjornfor** [comment]
@romildo: Great stuff! I just tested it on a laptop.

It seems the screensaver isn't working/installed:

```
An error occurred starting screensaver. Action
'activate' failed. Ensure you have xscreensaver
installed and running.
```

Off topic, is there a way to let window borders be "thicker" than one pixel in openbox? I'm having a hard time resizing windows with my mouse.

**Pull request merged**